packages/vc-verification/src/Ipfs-credential-resolver.ts

```diff
@@ -142,6 +142,10 @@ export class IpfsCredentialResolver implements CredentialResolver {
       await Promise.all(
         services.map(async ({ serviceEndpoint }) => {
           const claimToken = await this._ipfsStore.get(serviceEndpoint);
+          let offChainClaim;
```

Comment:
I don't see where it is used

Reply:
It stores the payload after decoding, which is followed after checking if the string fetched from IPFS is a JWT.

Reply:
and what happens with payload next?

Reply:
My bad, I forget to return it. The old code wasn't removed, will do it.